packages/big-design/src/components/Worksheet/hooks/useStore/useStore.ts

```diff
@@ -34,8 +33,8 @@ interface BaseState<Item> extends State {
   setTableRef: (ref: HTMLTableElement | null) => void;
 }
 
-export const createStore = () =>
-  create<BaseState<any>>((set) => ({
+export const createWorkSheetStore = <Item>() =>
```

Comment:
Maybe keep `Worksheet` for consistency?

```suggestion
export const createWorksheetStore = <Item>() =>
```

Reply:
agree, thanks!